modules/map/hdmap/hdmap_impl_test.cc

```diff
@@ -414,13 +416,17 @@ TEST_F(HDMapImplTestSuite, GetLocalMap) {
   std::pair<double, double> range{10.0, 10.0};
   ASSERT_EQ(0, hdmap_impl_.GetLocalMap(point, range, &local_map));
 
-  const std::string output_bin_file = FLAGS_output_dir + "/base_map.bin";
+  const std::string time_since_epoch = std::to_string(
+      std::chrono::steady_clock::now().time_since_epoch().count());
+  const std::string output_bin_file =
+      FLAGS_output_dir + "/base_map_" + time_since_epoch + " .bin";
```

Comment:
Please use `absl::StrCat` to avoid many temporary intermediate string objects.
https://github.com/ApolloAuto/apollo/blob/master/modules/map/hdmap/hdmap_util.cc#L43